cloud_infra_center/ocp_upi/README.md

```diff
@@ -0,0 +1,476 @@
+# IBM® Cloud Infrastructure Center
+
+IBM® Cloud Infrastructure Center is an advanced infrastructure management offering, including on-premises cloud deployments of IBM z/VM®-based and KVM based Linux® virtual machines on the IBM Z® and IBM LinuxONE platforms.
+
+More details: https://www.ibm.com/docs/en/cic/1.1.3
+
+# About this playbook
+
+The purposes of this playbook:
+
+1. Use IBM® Cloud Infrastructure Center to install Red Hat® OpenShift® Container Platform with user-provisioned infrastructure (UPI)
+
+2. Enhance the existing UPI process, such as 
+
+  - Rhcos images can be automatically downloaded and uploaded.
```

Comment:
These indentations in Markdown don't render correctly as bullets when you look at the README

Reply:
Thanks, I'll record and fix in another PR.